virtwho/virt/virt.py

```diff
@@ -352,11 +363,33 @@ def append_source_status_message(self, message):
         self._source_message += f"{message}."
 
     def append_destination_status_message(self, message):
-        self._destination_message = f"{message}."
+        self._destination_message += f"{message}."
 
     def set_destination(self, value):
         self._destination = value
 
+    def set_job_id(self, id):
```

Comment:
When `job_id` is property, then more pythonic way of implementing setter would be following:

```python
@job_id.setter:
def job_id(self, id):
    self._job_id = id
```